extensions/wikia/ImageReview/StatsDataGetter.class.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+class StatsDataGetter extends WikiaModel {
```

Comment:
No logic changes, all this code was pulled directly from the old `ImageReviewHelper` classs